lighty-core/lighty-parent/pom.xml

```diff
@@ -158,7 +158,7 @@
                 </plugin>
                 <plugin>
                     <artifactId>maven-checkstyle-plugin</artifactId>
-                    <version>3.1.2</version>
+                    <version>3.2.0</version>
                     <dependencies>
```

Comment:
Update checkstyle below to 9.3.

https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-checkstyle-plugin/3.2.0